client/layout/guided-tours/docs/API.md

```diff
@@ -260,9 +260,11 @@ Some examples:
 // detect state (has-thumbnail) and select a child (featured image)
 <Step target=".reader-post-card.has-thumbnail .reader-post-card__featured-image">
 
-// target id (notice the required space before #)
-<Step target=" #header">
+// target id
+<Step target="#header">
 
-// target element (again, space at the beginning)
+// target element
 <Step target=" body">
```

Comment:
Very good point! I wanted to document this very edge case situation, but it should be really clear that this is a hack and you should be solving the problem otherwise. Can you take a second look now? :)